src/app/core/models/issue.model.ts

```diff
@@ -48,6 +48,10 @@ export class Issue {
   teamChosenType?: string;
   teamAccepted?: boolean;
 
+  /** Fields for error messages during parsing of Github's issue description */
+  teamResponseError?: boolean;
+  testerResponseError?: boolean;
```

Comment:
Generally I believe errors should be None or Some. In this case it's more predictable to have explicit `false` mean no error rather than have the programmers account for cases where error might be `undefined`